src/libraries/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -697,11 +698,16 @@ public void WindowsSearchPatternWhitespace()
             Assert.Empty(GetEntries(TestDirectory, "\t"));
         }
 
-        [Fact]
+        [ConditionalFact]
         [PlatformSpecific(CaseSensitivePlatforms)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/51371", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
         public void SearchPatternCaseSensitive()
         {
+            if (RuntimeInformation.RuntimeIdentifier.StartsWith("iossimulator")
```

Comment:
Should we add a property to https://github.com/dotnet/runtime/blob/main/src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs and use it in the condition instead?

Reply:
Yes, good idea.